templates/configmap-env.yaml

```
@@ -15,6 +15,9 @@ data:
   DB_NAME: {{ .Values.postgresql.auth.database }}
   DB_POOL: {{ .Values.mastodon.sidekiq.concurrency | quote }}
   DB_USER: {{ .Values.postgresql.auth.username }}
+  {{- if .Values.mastodon.preparedStatements }}
```

```suggestion
  {{- if hasKey .Values.mastodon "preparedStatements" }}
```

this will ensure that if `.Values.mastodon.preparedStatements: false` this block will still be rendered. Alternatively, the default value in `values.yaml` could be `true` and then this `if` block could be removed.

---

Good point, honestly I'd prefer the latter option :+1: .

---

If you're setting it in `values.yaml` why even bother with checking if it's set?
Why not just quote the value and display it immediately?